src/js/view.js

```diff
@@ -0,0 +1,24 @@
+import { data } from "./model.js"
+
+
+export const render = () => {
+  const template = data.todos.map((item, index) => {
+    return `<li data-id="${index}" class="${item.checked ? 'completed' : 'false'}">
+              <div class="view">
+                <input class="toggle" type="checkbox" id="${item.id}" ${item.checked ? 'checked' : 'false'}/>
```

Comment:
```suggestion
                <input class="toggle" type="checkbox" id="${item.id}" ${item.checked ? 'checked' : ''}/>
```

checked 속성을 어떻게 다뤄야하는지 알아보기